test/runtests.jl

```diff
@@ -101,6 +101,66 @@ else  # run HiGHS tests
         @test r["ElectricStorage"]["size_kwh"] ≈ 83.3 atol=0.1
     end
 
+    @testset "Solar and ElectricStorage w/ zero ElectricStorage cost constants" begin
+        m1 = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false))
+        m2 = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false))
+        d = JSON.parsefile("./scenarios/pv_storage.json");
+        
+        d["ElectricStorage"]["installed_cost_constant"] = 0
+        d["ElectricStorage"]["replace_cost_constant"] = 0
+        d["ElectricStorage"]["cost_constant_replacement_year"] = 10
+    
+        d["Settings"] = Dict{Any,Any}("add_soc_incentive" => false)
```

Comment:
@toddleif out of curiosity why did you include setting the add_soc_incentive to false?

Reply:
That's a good point. I removed that line, which will keep add_soc_incentive as the default, true.